deployment/aws-batch/job-definitions/production-pfb-analysis-run-job.json

```diff
@@ -15,8 +15,16 @@
             "host": {
                 "sourcePath": "/media/nvme0n1/pgdata"
             }
+        }, {
+            "name": "tmp",
+            "host": {
+                "sourcePath": "/media/nvme0n1/tmp"
+            }
         }],
         "environment": [{
+            "name": "NB_MAX_TRIP_DISTANCE",
+            "value": "2680"
```

Comment:
Yeah, that would work as well.  But the setup I'm proposing wouldn't cause inconsistencies, I don't think.  
```
NB_MAX_TRIP_DISTANCE="${NB_MAX_TRIP_DISTANCE:-2680}"
NB_BOUNDARY_BUFFER="${NB_BOUNDARY_BUFFER:-$NB_MAX_TRIP_DISTANCE}"
```
would mean "set `NB_MAX_TRIP_DISTANCE` to the value in the environment, but fall back to 2680, then set `NB_BOUNDARY_BUFFER` to the value in the environment but fall back to whatever `NB_MAX_TRIP_DISTANCE` is set to."  So if either is already set in the environment that value will be used.

However, having lots of different component scripts do their own setup like this probably doesn't make sense.  It seems like its purpose would be to allow them to operate independently, but that's not something we need and it probably wouldn't work out that well anyway given all the development that has taken place treating it as a whole.

So yeah, defining variables further out and `export`ing them sounds fine to me, it just doesn't fit the existing pattern as well.